anaconda.spec.in

```diff
@@ -279,8 +279,8 @@ Requires: nvme-cli
 Requires: podman
 
 %description install-img-deps
-The anaconda-install-img-deps metapackage lists all boot.iso installation
-image dependencies. Add this package to an image build (eg. with lorax) to
+The anaconda-install-img-deps meta-package lists all boot.iso installation
```

Comment:
This is also mentioned in many documentations as one word.
https://help.ubuntu.com/community/MetaPackages

I would prefer to mark the failure as expected and silence it.